ocp_resources/datavolume.py

```diff
@@ -212,14 +217,20 @@ def scratch_pvc(self):
             client=self.client,
         )
 
+    def _dv_phase(self):
+        if self.exists:
```

Comment:
`return self.instance.status.phase if self.exists else self.exists`

No need to have `DV_DOES_NOT_EXISTS` if it's just for status (not used to log for example)

Reply:
if phase don't exists the return value is none, I want to have different return value for DV don't exists and phase in DV don't exists

Reply:
where do you use the value?
The else is not needed

(Please do not resolve the conversion if you only answer my comment, this is ongoing conversion)

Reply:
in line 234 
if sample in [self.Status.PENDING, None]
check if the value is None, if the phase is none continue, but if the dv does not exists break

in line 271
if sample in [self.Status.SUCCEEDED, DV_DOES_NOT_EXISTS]
I want to catch the case that dv phase is succeeded/dv do not exists (which indicate that dv succeeded) 

(of course I am not gonna resolve it until we agree, this was the only conversation I didn't resolved yet :))